code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -78,10 +78,41 @@
 	speech_args[SPEECH_MESSAGE] = trim(message)
 
 /obj/item/clothing/mask/joy
-	name = "joy mask"
-	desc = "Express your happiness or hide your sorrows with this laughing face with crying tears of joy cutout."
+	name = "Emotional Mask"
+	desc = "Express your happiness or hide your sorrows with this modular cutout."
 	icon_state = "joy"
+	clothing_flags = ALLOWINTERNALS
+	resistance_flags = FIRE_PROOF //let us have this i beg you, it's a medium size breathmask :(
```

Comment:
No need for fireproofness for a literal papermask

Reply:
:(